api/lib/source/cosmosV0-source.js

```diff
@@ -359,10 +359,13 @@ class CosmosV0API extends RESTDataSource {
     const balances = balancesResponse || []
     // the user might not have liquid staking tokens but have staking tokens delegated
     // if we don't add the staking denom, we would show a 0 total for the staking denom which is wrong
-    if (delegations.length > 0 && !balancesResponse.find(({denom}) => denom === this.network.stakingDenom)) {
```

Comment:
@Bitcoinera this is still here?

Reply:
Hey. The code is the same, I reverted the fix, but if you merge with `develop` the fix should be there

Reply:
ah, you need to do it anyway because of the conflict